README.md

```diff
@@ -84,8 +84,6 @@ export sourceid=lib1
 java -jar client/target/mod-reservoir-client-fat.jar \
   --source $sourceid \
   --xsl xsl/marc2inventory-instance.xsl \
```

Comment:
Change this to use `localid.xsl` stylesheet. We don't need Inventory records anymore for this example.

Reply:
Sure, but I am looking at that stylesheet  and it contains shitload of inventory stuff still. What's worse mod-reservoir is quite married internal to the inventory/original stuff. We did a few tricks to get this to work.

Reply:
`localid.xsl` does not contain anything inventory related.

Yes, the client should be cleaned up, but that's a seperate ticket. I'd like to be able to specify the localid without the stylesheet for example. And if we keep xsl support it should use the XSL3.0 json xml encoding and not the homegrown stuff.

Reply:
I assume localid.xsl does not exist. So I've created one.

Reply:
> I assume localid.xsl does not exist. So I've created one.

oh. It did exist, and so there's a smaller version now which does not make a deep copy.

Reply:
I forgot XSL so i thought a deep copy was the only option to send input to output. I guess not.